domains/cunny.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "terratails",
+        "email": "glenpaulchoco@yahoo.com"
+    },
+    "record": {
+        "CNAME": "messier.cunny"
```

Comment:
i dont think .cunny is a tld so site is invalid, please put another record in CNAME